src/style.css

```diff
@@ -0,0 +1,11 @@
+body {
+    /* background-color: #F1EFEF; */
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Expected indentation of 2 spaces (indentation)](https://app.codacy.com/gh/WebHeroSchool/todo_test/pullRequest?prid=5693637)